apps/redi-connect/src/components/organisms/ApplicationCard.tsx

```diff
@@ -144,23 +137,23 @@ const ApplicationCard = ({
             <Content>{application.expectationText}</Content>
           </>
         )}
-        {currentUserIsMentor && application.status === 'applied' ? (
+        {currentUserIsMentor && application.status === 'applied' && (
           <>
             <ConfirmMentorship
               match={application}
               hasReachedMenteeLimit={hasReachedMenteeLimit}
             />
             <DeclineMentorshipButton match={application} />
           </>
-        ) : null}
+        )}
```

Comment:
The shortcut is "faster" and less verbose than a ternary. And considering the size of the components that you are building (that would be another topic), being more concise would be always better. 

I understand why ternary is suggested to be used in this "Tao", but in the end, it's an opinion of a developer and the only given argument is a bad return that, in case of being done with a ternary, leads to a 50% of chance to match the expected behavior. So for me, the ternary is a trap; is a way of hiding a possible error inside a poka-yoke. And considering that unit testing has not been a priority for the design of the components, it's way better to have a weird return of a `0` or whatever so evident wrong thing to be rendered that can be noticed than a rendered-or-not component based on a not-so-transparent behavior. A ternary is ticking yourself in solitaire.